adr/023-deployment-slots.md

```diff
@@ -0,0 +1,40 @@
+# 23. Deployment Slots for Zero Downtime Deploys
```

Comment:
I'd like to see a reason why we are using deployment slots even though there are some significant negative impacts and risks.

Reply:
Basically the other options were even worse - Kubernetes, roll our own, or switch to AWS (or abandon zero-downtime deploys). We can add a mention of the alternatives